CodeGeneration/Sources/SyntaxSupport/StmtNodes.swift

```diff
@@ -261,7 +261,7 @@ public let STMT_NODES: [Node] = [
 
   // fallthrough-stmt -> 'fallthrough' ';'?
   Node(
-    kind: .fallthroughStmt,
+    kind: .fallThroughStmt,
```

Comment:
Interesting. I prefer capitalized for all but `fallthrough`, even though "fallthrough" isn't a word 🤔. Suppose this makes sense then 👍 (if we keep `associatedType` and `typeAlias`)

Reply:
Yes, for consistency, let’s keep `T` capitalized here as well.